lib/services/messenger-types.d.ts

```diff
@@ -42,32 +43,47 @@ export interface ReceiptIds extends MessengerBaseIds {
 	message: string;
 	/** The ID of the thread. */
 	thread: string;
+	// https://github.com/resin-io-modules/resin-procbots/issues/301
+	/**
+	 * For Intercom messages on Front, whether they had an empty subject line
+	 * An empty subject line indicates the message was inputted via the Front or Intercom UI
+	 * A subject line indicates the message was inputted via the Front API
+	 * Needed because Front mangles the message into plaintext and refuses to let you see raw
```

Comment:
Sigh. :(

Reply:
It'll be better in Jellyfish.  At the moment I'm integrating quirky with quirky via quirky, which is a big pain.  Under Jellyfish I'll at least be integrating quirky with quirky via **something we control**.